data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg

```diff
@@ -130,22 +130,22 @@
 
         [message]
             speaker=Hamel
-            message= _ "Oh, there are orcs enough down here, but ye’ll not likely see them; they hide from us like vermin. You see, lad, in these caves there is no more fearsome a foe than a dwarf. Not only do we move through the caves swiftly, but we know them like the backs of our hands. The orcs did indeed manage to swamp our defenses with their sheer numbers all those long years ago, but not before we surprised and slew their leader — the cursed Khazg Black-Tusk. Since then, although the orcs still foul these caves with their stench, their attacks have been uncoordinated and erratic, wi’ the orcs as likely to attack each other as they are to attack us."
+            message= _ "Oh, there are orcs enough down here, but ye’ll not likely see them; they hide from us like vermin. You see, lad, in these caves there is no more fearsome a foe than a dwarf. Not only do we move through the caves swiftly, but we know them like the backs of our hands. All those years ago, the orcs did indeed manage to swamp our defenses with their sheer numbers, but not before we surprised and slew their leader — that cursed Khazg Black-Tusk. Since then, although the orcs still foul these caves with their stench, their attacks have been uncoordinated and erratic, wi’ the orcs as likely to attack each other as they are to attack us."
         [/message]
 
         [message]
             speaker=Hamel
-            message= _ "In more recent years, however, I ha’ noticed that there have been fewer orcs and more skeletons around. The orcs seem to fear them and shy away from them whenever possible, but their troll allies are another matter. Yon trolls actually seem to take great pleasure out of crushing the skeletons to the dust and sucking any marrow that remains in their bones..."
+            message= _ "In more recent years, however, I ha’ noticed that there have been fewer orcs and more skeletons around. The orcs seem to fear them and shy away from them whenever possible, but their troll allies are another matter. Yon trolls actually seem to take great pleasure out of crushing the skeletons to the dust and sucking what marrow remains in their bones."
```

Comment:
I like the new one on this.

It's a small change in a long sentence, so having a translators' changelog would be helpful. An idea for how to do that was to have a line in the Git commit (as it's only useful to someone changing from old to new text) saying what changed.

Alternatively, po comments here in the WML file might be good for changes to strings that have been unchanged for ages (and this text seems to be last changed in 7d9ef83560f74c2258598337b6b842ddfa657c0f, 13 years ago).

Reply:
The trouble is that different people have views of what's a "big" change. The main translation tool that works on Windows, Poedit, doesn't have diff highlighting (yet), and so translators end up comparing the text in the "Source text" pane with the "Previous source text" pane by hand. The proper fix is to get better translation tools, but at least for the moment I think this image explains why I'm being so hesitant about changes adding or removing commas.

![image](https://github.com/wesnoth/wesnoth/assets/101462/0b72744e-4d7f-45b1-9230-5b2319296099)

Reply:
I've done it in all the other PRs, and they got merged. So... Maybe you just assume that the entire sentence need's translated?